packages/@tests/marine/node/src/index.ts

```diff
@@ -0,0 +1 @@
+export const index = () => {};
```

Comment:
why is this necessary?

Reply:
oh, my. I even forgot to uncomment these tests :)

Reply:
Removed this file completely